### PR TITLE
Add MPLS L3VPN manual linux data path config 

### PR DESCRIPTION
**Pull request by louberger**
This is known to work on ubuntu17 4.13.0-31 - and is likely to fail on older kernels...

### CODE REVIEW

File: bgp_l3vpn_to_bgp_vrf/customize.py [hunk 1]
```diff
@@ -133,26 +133,30 @@ def doCmd(tgen, rtr, cmd):
...
     for intf in intfs:
         doCmd(tgen, 'r2', 'echo 1 > /proc/sys/net/mpls/conf/{}/input'.format(intf))
-    #configure MPLS
+    #configure VRFs & MPLS
     rtrs = ['r1', 'r3', 'r4']
-    cmds = ['echo 1 > /proc/sys/net/mpls/conf/lo/input']
+    cmds = ['ip link add cust1 type vrf table 10',
+            'ip ru add oif cust1 table 10',
+            'ip ru add iif cust1 table 10',
+            'ip link set dev cust1 up']
```
Comment (pguibert6WIND) [inline bgp_l3vpn_to_bgp_vrf/customize.py]: Hi @louberger ,
Will it be possible to pass an option to configure ( or not for now) the VRF Lite.
I think it brings added value to customize the test by runing it in both modes: vrf lite or vrf netns

What is sure is that if VRF backend is netns, those commands will not have to be executed
What is not sure yet, is if VRF netns should be done here, or earlier ( I will keep you in touch).

**Comment by louberger** [comment]
Perhaps just create a whole new test for net NS?

**Comment by louberger** [comment]
@mwinter-osr these failures look like environment issues, specifically that the latest iproute2 isn't being used.  I'll put in a workaround to skip the failing tests when the commands aren't present, but would be better to have them execute in the ci system...

**Comment by louberger** [comment]
@pguibert6WIND requested name changes just pushed top this branch

**Comment by louberger** [comment]
@mwinter-osr @rzalamena this is ready to be merged once it passes CI

**Pull request merged**